lua/entities/gmod_wire_expression2/core/custom/wiring.lua

```diff
@@ -108,7 +108,17 @@ __e2setcost(5)
 e2function wirelink entity:wirelink()
 	if not IsValid(this) then return self:throw("Invalid entity!", nil) end
 	if not isOwner(self, this) then return self:throw("You do not own this entity!", nil) end
-	
+
+	return this
+end
+
+e2function wirelink entity:createWirelink()
```

Comment:
What's the point of this function?

Reply:
Mimics the old `entity:wirelink()` that would create a wirelink output. You can do the same with the Wire tool, but I guess some people wanted to generate wirelinks using E2 instead.

Reply:
Why not add back the behavior to entity:wirelink instead of adding this new function?

Reply:
I don't see it as necessary. You don't create entity outputs when doing `entity`. I don't believe E2 users are using the wirelink output since they just used `entity:wirelink()` instead.